### PR TITLE
Install Bower, Grunt CLI and Gulp and bump versions

### PR DESCRIPTION
**Pull request by cyrille-leclerc**
Install Bower, Grunt CLI and Gulp and bump versions

### CODE REVIEW

**Comment by cyrille-leclerc** [comment]
cc @ndeloof @ydubreuil @michaelneale

File: Dockerfile [hunk 1]
```diff
@@ -127,28 +126,38 @@ RUN mkdir -p /home/jenkins/.local/bin/ \
   && chown -R jenkins:jenkins /home/jenkins/.local
 
 #====================================
+# NODE JS
+# See https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions
+#====================================
+RUN curl -sL https://deb.nodesource.com/setup_4.x | bash \
```
Comment (ndeloof) [inline Dockerfile]: as you can run npm you already have node installed.

Comment (cyrille-leclerc) [inline Dockerfile]: @ndeloof why do you say I can run `npm`? I run it later in the script.

Comment (ndeloof) [inline Dockerfile]: my bad, didn't read carrefully enough

Comment (cyrille-leclerc) [inline Dockerfile]: Do I have your :+1: ? :-)

**Comment by ndeloof** [comment]
LGTM :+1: :bee: etc

**Pull request merged**